.env.example

```
@@ -0,0 +1,18 @@
+# Copy this file to .env and fill in the values before running docker-compose.
+
+# ── PostgreSQL ─────────────────────────────────────────────────────────────
+# These are used both by the db service and to build the connection URL for web.
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=changeme
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)